packages/block-library/src/post-featured-image/editor.scss

```diff
@@ -25,3 +25,10 @@ div[data-type="core/post-featured-image"] {
 		}
 	}
 }
+
+.block-library-post-featured-image-dimension-controls {
```

Comment:
Interesting that we need these styles, I guess the ideal solution would be to have another parent `Flex` or something else that does this automatically without CSS. That's fine though for now.

Reply:
Yeah.. It seems that we mostly put components with `BaseControl` built in, that has these styles.